test/splitter.js

```diff
@@ -97,34 +89,61 @@ contract('Splitter', async accounts => {
         checkEventNotEmitted();
     });
 
+    it("splitDeposit is pausable", async () => {
+
+        splitter.pause({from: contractOwner});
+
+        await truffleAssert.reverts(
+            splitter.splitDeposit(recipientTwoAddress, recipientOneAddress,{from: senderAddress, value: 4}),
+            "Pausable: paused"
+        );
+
+        const senderOwed = await splitter.balances(senderAddress);
+        const recipientOneOwed = await splitter.balances(recipientOneAddress);
+        const recipientTwoOwed = await splitter.balances(recipientTwoAddress);
+
+        const expected = toBN(0).toString(10);
+
+        assert.strictEqual(senderOwed.toString(10), expected);
+        assert.strictEqual(recipientOneOwed.toString(10), expected);
+        assert.strictEqual(recipientTwoOwed.toString(10), expected);
+
+        splitter.unpause({from: contractOwner});
```

Comment:
Await?
Also, it is better to test the `pause` in an `it`, and `resume` in another `it`.